### PR TITLE
Add Sawtooth Explorer project to repository

### PR DESCRIPTION
**Pull request by GiniH**
This merge request adds the entire project to the repository.

Views included are the Explorer which enables navigation between transaction, batch, and block entities, standalone detail views of those listed entities, and the State Monitor page which watches for state deltas given a list of address prefixes.

The README has been updated with instructions for how to setup and run the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,82 @@
...
+SawtoothExplorer is built using [Angular](https://angular.io/) and
+[Angular Material](https://material.angular.io/). The project is managed using
+the [Angular CLI](https://cli.angular.io/).
+
+## Getting Started
+Before starting the SawtoothExplorer app, have a local validator running.
+Instructions for doing so are available in the Sawtooth documentation,
+under [Installing and Running Sawtooth](https://sawtooth.hyperledger.org/docs/core/releases/latest/app_developers_guide/installing_sawtooth.html).
+
+Once this is running, the SawtoothExplorer app must be configured to
```
Comment (kidrecursive) [inline README.md]: The REST API is what the app talks to, not the validator node

File: package.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "dok-chain-ui",
+  "version": "0.0.0",
+  "license": "MIT",
```
Comment (dcmiddle) [inline package.json]: Can this be made Apache 2.0?

Comment (GiniH) [reply package.json]: Absolutely, will update this and make the requested licensing additions.

**Comment by rjones-lf** [comment]
If you could do a squash commit before merging, that would be great.

This is so the license status of the files in the intermediary state won't be in question.

**Comment by vaporos** [comment]
Is "Copyright 2017 Intel Corporation" on the files correct? Isn't this a Pokitdok contribution?

**Comment by dcmiddle** [comment]
Depending on who authored the files please do attribute the copyright correctly as @vaporos pointed out. 
In the future if you add text or image files you'll want to license those as 
https://creativecommons.org/licenses/by/4.0/

Thanks!

**Comment by GiniH** [comment]
Fixed copyright attributions and re-squashed in the latest, though wondering about the image attributions as you mentioned @dcmiddle -- they're all just different formats of the Sawtooth logo. How would you recommend I handle the license file for that directory?

**Comment by dcmiddle** [comment]
@GiniH that sounds good.
@cianx I think the ball is in your court to merge.

**Pull request merged**